workspaces/ui-v2/src/optic-components/pages/diffs/CapturePage.tsx

```diff
@@ -24,19 +28,29 @@ import {
   useDiffUndocumentedUrlsPageLink,
 } from '../../navigation/Routes';
 import { EndpointName } from '../../common';
-import { useHistory } from 'react-router-dom';
 import ApproveAll from '../../diffs/render/ApproveAll';
 import { useCaptures } from '../../hooks/useCapturesHook';
 
-export function CapturePage(props: { showDiff?: boolean }) {
+function CapturePage(props: { showDiff?: boolean }) {
+  const { boundaryId, environment } = useParams<{
+    boundaryId?: string;
+    environment?: string;
+  }>();
   const capturesState = useCaptures();
   const history = useHistory();
   const diffEnvironmentsRoot = useDiffEnvironmentsRoot();
 
-  const noCaptures =
-    !capturesState.loading && capturesState.captures.length === 0;
+  const noCaptures = capturesState.captures.length === 0;
+  // The only current environment supported is local
+  const isValidEnvironment = environment === 'local';
+  // TODO this currently does not get reached if we have an invalid boundary,
```

Comment:
https://github.com/opticdev/optic/blob/04bcab1e719b100e79ce36e06509ee6664b6fdef/workspaces/ui-v2/src/optic-components/pages/diffs/ReviewDiffPages.tsx#L33 this part errors if in the url we specify an incorrect / invalid diff.

Only possible if the user enters the url manually - I left this for now, and I think this can kind of fit into better error handling

Reply:
agreed on the approach here.